pipeline/init.bash

```diff
@@ -16,16 +15,17 @@ export CK8S_ENVIRONMENT_NAME="${CK8S_ENVIRONMENT_NAME:-apps-${CK8S_CLOUD_PROVIDE
 
 # Update ck8s configuration
 
-objectStoreProvider=$(sops_exec_file "${secrets[secrets_file]}" 'yq r -e {} objectStorage.type')
+objectStoreProvider="s3"
```

Comment:
This might seem silly for now, but it makes it very easy to add some logic where we want to use gcs for when `CK8S_CLOUD_PROVIDER == "google"` or something like that